openfisca_uk_data/datasets/frs/frs.py

```diff
@@ -154,7 +154,7 @@ def add_personal_variables(frs: h5py.File, person: DataFrame):
     # Add basic personal variables
     age = person.AGE80 + person.AGE
     frs["age"] = age
-    frs["role"] = np.where(person.AGE80 == 0, "adult", "child").astype("S")
+    frs["role"] = np.where(person.AGE80 == 0, "child", "adult").astype("S")
```

Comment:
Could you add a comment on what `role` and `AGE80` are?

Reply:
Added

Reply:
Thanks, just to be explicit could you add one more sentence (if I'm understanding it correctly):
```
# AGE80 == 0 therefore indicates that the record is from the child table.
```

Reply:
Ah no problem, will add that